test/generator/memory_profiler_mandelbrot_aottest.cpp

```diff
@@ -82,7 +82,9 @@ int main(int argc, char **argv) {
     printf("argmin expected value\n  stack peak: %d\n", argmin_stack_peak);
     printf("\n");
 
-    halide_do_par_for(nullptr, launcher_task, 0, num_launcher_tasks, nullptr);
```

Comment:
launcher_task unconditionally returns zero, so this call also unconditionally returns zero, and then check is just confusing.

Reply:
Commented.